### DISCUSSION TITLE
Momentary/Toggle Layer on one key

### DISCUSSION THREAD

**Discussion by snaotheus** [KinesisCorporation/Adv360-Pro-ZMK#489] [Q&A]
Is there a way to have one key switch layer momentarily when pressed and held, but toggle to another layer when tapped (or maybe double tapped)? Kind of like &lt, except that the behavior on tap is &tog instead of a keypress output.

**Comment by ReFil** [comment]
Yep, just create a custom hold-tap https://zmk.dev/docs/behaviors/hold-tap

**Reply by snaotheus** [reply]
Can that be done in the graphical editor, or does that have to be modified directly in the source files?
![image](https://github.com/KinesisCorporation/Adv360-Pro-ZMK/assets/8275907/d67e4183-31f9-46b5-91da-20709c856247)
